src/Raven.Studio/wwwroot/manifest.json

```diff
@@ -1,18 +1,31 @@
 {
-    "name": "RavenDB",
+    "name": "Raven Studio",
```

Comment:
I wonder if our studio could have some node name in it's name. What if you have cluster - then you end up with few apps with the same name. Btw. did you check if you can install more than one app?

Reply:
I'm aware of this limitation, but it means we'd need to generated the manifest on the server side. I decided against adding that in this first iteration, but I can add it.

Reply:
And yeah, you can install multiple Raven Studios.

Reply:
Marcin, to do this, we need to generate the manifest on the server. To do that, we could either:

- Leave the manifest.json file in the solution, but when its requested, we read it, change the name, then return it.
- Delete manifest.json file and just generate it on the fly from the server.

Do you have a preference?

Reply:
Actually, it looks like we can [generate the manifest dynamically on the client](https://medium.com/@alshakero/how-to-setup-your-web-app-manifest-dynamically-using-javascript-f7fbee899a61). So this is option 3.